data/subdivision-codes.csv

```diff
@@ -1,4682 +1,4682 @@
-AO,MOX,Moxico,Province
-AO,NAM,Namibe,Province
-AO,UIG,Uíge,Province
-AO,ZAI,Zaire,Province
-AR,A,Salta,Province
-AR,B,Buenos Aires,Province
-AR,C,Ciudad Autónoma de Buenos Aires,City
-AR,D,San Luis,Province
-AR,E,Entre Ríos,Province
-AR,F,La Rioja,Province
-AR,G,Santiago del Estero,Province
-AR,H,Chaco,Province
-AR,J,San Juan,Province
-AR,K,Catamarca,Province
-AR,L,La Pampa,Province
-AR,M,Mendoza,Province
-AR,N,Misiones,Province
-AR,P,Formosa,Province
-AR,Q,Neuquén,Province
-AR,R,Río Negro,Province
-AR,S,Santa Fe,Province
-AR,T,Tucumán,Province
-AR,U,Chubut,Province
-AR,V,Tierra del Fuego,Province
-AR,W,Corrientes,Province
-AR,X,Córdoba,Province
-AR,Y,Jujuy,Province
-AR,Z,Santa Cruz,Province
-AT,1,Burgenland,state
-AT,2,Kärnten,state
-AT,3,Niederösterreich,state
-AT,4,Oberösterreich,state
-AT,5,Salzburg,state
-AT,6,Steiermark,state
-AT,7,Tirol,state
-AT,8,Vorarlberg,state
-AT,9,Wien,state
-AU,ACT,Australian Capital Territory,Territory
-AU,NSW,New South Wales,State
-AU,NT,Northern Territory,Territory
-AU,QLD,Queensland,State
-AU,SA,South Australia,State
-AU,TAS,Tasmania,State
-AU,VIC,Victoria,State
-AU,WA,Western Australia,State
-AZ,ABS,Abşeron,Rayon
-AZ,AGA,Ağstafa,Rayon
-AZ,AGC,Ağcabədi,Rayon
-AZ,AGM,Ağdam,Rayon
-AZ,AGS,Ağdaş,Rayon
-AZ,AGU,Ağsu,Rayon
-AZ,AST,Astara,Rayon
-AZ,BA,Bakı,Municipality
-AZ,BAB,Babək,Rayon
-AZ,BAL,Balakən,Rayon
-AZ,BAR,Bərdə,Rayon
-AZ,BEY,Beyləqan,Rayon
-AZ,BIL,Biləsuvar,Rayon
-AZ,CAB,Cəbrayıl,Rayon
-AZ,CAL,Cəlilabad,Rayon
-AZ,CUL,Culfa,Rayon
-AZ,DAS,Daşkəsən,Rayon
-AZ,FUZ,Füzuli,Rayon
-AZ,GA,Gəncə,Municipality
-AZ,GAD,Gədəbəy,Rayon
-AZ,GOR,Goranboy,Rayon
-AZ,GOY,Göyçay,Rayon
-AZ,GYG,Göygöl,Rayon
-AZ,HAC,Hacıqabul,Rayon
-AZ,IMI,İmişli,Rayon
-AZ,ISM,İsmayıllı,Rayon
-AZ,KAL,Kəlbəcər,Rayon
-AZ,KAN,Kǝngǝrli,Rayon
-AZ,KUR,Kürdəmir,Rayon
-AZ,LA,Lənkəran,Municipality
-AZ,LAC,Laçın,Rayon
-AZ,LAN,Lənkəran,Rayon
-AZ,LER,Lerik,Rayon
-AZ,MAS,Masallı,Rayon
-AZ,MI,Mingəçevir,Municipality
-AZ,,Naftalan,Municipality
-AZ,NEF,Neftçala,Rayon
-AZ,NV,Naxçıvan,Municipality
-AZ,NX,Naxçıvan,Autonomous republic
-AZ,OGU,Oğuz,Rayon
-AZ,ORD,Ordubad,Rayon
-AZ,QAB,Qəbələ,Rayon
-AZ,QAX,Qax,Rayon
-AZ,QAZ,Qazax,Rayon
-AZ,QBA,Quba,Rayon
-AZ,QBI,Qubadlı,Rayon
-AZ,QOB,Qobustan,Rayon
-AZ,QUS,Qusar,Rayon
-AZ,SA,Şəki,Municipality
-AZ,SAB,Sabirabad,Rayon
-AZ,SAD,Sədərək,Rayon
-AZ,SAH,Şahbuz,Rayon
-AZ,SAK,Şəki,Rayon
-AZ,SAL,Salyan,Rayon
-AZ,SAR,Şərur,Rayon
-AZ,SAT,Saatlı,Rayon
-AZ,SBN,Şabran,Rayon
-AZ,SIY,Siyəzən,Rayon
-AZ,SKR,Şəmkir,Rayon
-AZ,SM,Sumqayıt,Municipality
-AZ,SMI,Şamaxı,Rayon
-AZ,SMX,Samux,Rayon
-AZ,SR,Şirvan,Municipality
-AZ,SUS,Şuşa,Rayon
-AZ,TAR,Tərtər,Rayon
-AZ,TOV,Tovuz,Rayon
-AZ,UCA,Ucar,Rayon
-AZ,XA,Xankəndi,Municipality
-AZ,XAC,Xaçmaz,Rayon
-AZ,XCI,Xocalı,Rayon
-AZ,XIZ,Xızı,Rayon
-AZ,XVD,Xocavənd,Rayon
-AZ,YAR,Yardımlı,Rayon
-AZ,YE,Yevlax,Municipality
-AZ,YEV,Yevlax,Rayon
-AZ,ZAN,Zəngilan,Rayon
-AZ,ZAQ,Zaqatala,Rayon
-AZ,ZAR,Zərdab,Rayon
-BA,BIH,Federacija Bosne i Hercegovine,Entity
-BA,BRC,Brčko distrikt,District with special
-status
```

Comment:
@gradedSystem  can you please check this issue? I think I had solved by merging with the previous line

Reply:
@sabas can you take a look I added a function to the `scripts/prepare.py`, tried merging previous line in the .sh script but wasn't be able to figure out

Reply:
that works thanks :)